@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1324,6 +1324,10 @@ export class TextElementsRenderer {
         const tempMeasurementParams: MeasurementParameters = {};
         const tempBufferAdditionParams: TextBufferAdditionParameters = {};
 
+        const tileGeometryManager = this.m_mapView.tileGeometryManager;
+        const hiddenKinds =
```

Comment:
```js
const hiddenKinds = tileGeometryManager && tileGeometryManager.hiddenGeometryKinds;
```

Will return `tileGeometryManager.hiddenGeometryKinds` if `tileGeometryManager !== undefined`;
Otherwise returns `undefined`;

Unless there is no specific need in typescript check 🤔

Reply:
Please read our coding style: Avoid truthies